src/components/input-elements/InputText.vue

```diff
@@ -39,29 +59,103 @@ export default {
 </script>
 
 <style scoped>
+.capp-input__wrapper {
+  width: 100%;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  gap: 1rem;
+}
+.capp-input__wrapper span {
+  width: 100%;
+}
 .capp-label__default {
   display: block;
   color: var(--primary-dark);
-  font-weight: bold;
-  padding-top: 0.5rem;
-  padding-bottom: 0.2rem;
+
+  padding-block: calc(var(--s-font) / 2);
+  font-size: 1.5rem;
+  letter-spacing: 0.1rem;
 }
 
 .capp-input__default {
   display: block;
+  width: 100%;
   border: 2px solid var(--clr-trans);
   border-radius: 0.5rem;
-  padding: 0.5rem;
+  padding-inline: 0.5rem;
+  padding-block: 1rem;
+  font-size: var(--m-font);
   background: var(--clr-bg-main);
   box-shadow: inset 0 5px 5px -2px var(--secondary-dark);
   color: var(--font-color-dark);
 }
-
+.capp-input__default:hover {
+  cursor: pointer;
+}
 .capp-input__default::placeholder {
-  opacity: 0.8;
+  opacity: 0.5;
 }
 
 .capp-input__default:focus {
   outline-color: var(--primary-light);
 }
+.capp-input__invalid-input {
+  color: var(--error-color);
+  font-size: var(--s-font);
+  width: 100%;
+  height: max-content;
+  text-align: right;
+  padding-right: calc(var(--s-font) / 2);
+  margin-top: calc(var(--s-font) / 1.5);
+}
+.input__valid {
+  color: transparent;
+}
+.input__invalid {
+  color: var(--error-color);
+}
+
+/*=====================*/
+.capp-input__help-wrapper {
+  position: relative;
+}
+.capp-input__btn {
+  all: unset;
+  width: calc(var(--s-font) * 2);
+  aspect-ratio: 1;
+}
+.capp-input__btn:active > svg {
+  fill: var(--primary-mid);
+}
+.capp-input__btn svg {
+  fill: var(--secondary-default);
+  position: relative;
+}
+
+.capp-input__help {
+  position: absolute;
+  top: 20p;
+  left: 0;
+  display: block;
+  font-size: 1.2rem;
+  background: var(--secondary-light);
+  border: 1px solid var(--primary-dark);
+  border-radius: 1rem;
+  width: 20rem;
+  height: auto;
+  padding: 1rem;
+  text-align: start;
+}
+/* .capp-input__help::after {
```

Comment:
Benötigen wir den Auskommentierten Text noch in der Zukunft?
Ansonsten würde ich den rauslöschen, damit unser Code übersichtlich bleibt.

Reply:
ja der kann raus , hat wohl vergessen zu löschen